CCD-support/Multitenancy/capi_capm3-mult-instances/01-vm-setup.sh

```diff
@@ -0,0 +1,243 @@
+nmcli con load /etc/NetworkManager/system-connections/baremetal.nmconnection
+nmcli con up baremetal
+
+cat <<EOF > node-1.xml
+<domain type='kvm'>
+	<name>node-1</name>
+	<memory unit='MiB'>8000</memory>
+	<vcpu>2</vcpu>
+	<os>
+		<type arch='x86_64' machine='q35'>hvm</type>
+		<boot dev='network'/>
+		<bootmenu enable='no'/>
+	</os>
+	<features>
+		<acpi/>
+		<apic/>
+		<pae/>
+	</features>
+	<cpu mode='host-passthrough'/>
+	<clock offset='utc'/>
+	<on_poweroff>destroy</on_poweroff>
+	<on_reboot>restart</on_reboot>
+	<on_crash>restart</on_crash>
+	<devices>
+		<disk type='volume' device='disk'>
+			<driver name='qemu' type='qcow2' cache='unsafe'/>
+			<source pool='mypool' volume='node-1.qcow2'/>
+			<target dev='sda' bus='scsi'/>
+		</disk>
+		<controller type='scsi' model='virtio-scsi' />
+		<interface type='bridge'>
+			<mac address='00:5c:52:31:3a:9c'/>
+			<source bridge='provisioning-1'/>
+			<model type='virtio'/>
+		</interface>
+		<interface type='bridge'>
+			<mac address='00:5c:52:31:3b:9c'/>
+			<source bridge='baremetal'/>
+			<model type='virtio'/>
+		</interface>
+		<console type='pty'/>
+		<input type='mouse' bus='ps2'/>
+		<graphics type='vnc' port='-1' autoport='yes'/>
+		<video>
+			<model type='cirrus' vram='9216' heads='1'/>
+		</video>
+	</devices>
+</domain>
+EOF
+cat <<EOF > node-2.xml
+<domain type='kvm'>
+	<name>node-2</name>
+	<memory unit='MiB'>8000</memory>
+	<vcpu>2</vcpu>
+	<os>
+		<type arch='x86_64' machine='q35'>hvm</type>
+		<boot dev='network'/>
+		<bootmenu enable='no'/>
+	</os>
+	<features>
+		<acpi/>
+		<apic/>
+		<pae/>
+	</features>
+	<cpu mode='host-passthrough'/>
+	<clock offset='utc'/>
+	<on_poweroff>destroy</on_poweroff>
+	<on_reboot>restart</on_reboot>
+	<on_crash>restart</on_crash>
+	<devices>
+		<disk type='volume' device='disk'>
+			<driver name='qemu' type='qcow2' cache='unsafe'/>
+			<source pool='mypool' volume='node-2.qcow2'/>
+			<target dev='sda' bus='scsi'/>
+		</disk>
+		<controller type='scsi' model='virtio-scsi' />
+		<interface type='bridge'>
+			<mac address='00:5c:52:31:3a:ad'/>
+			<source bridge='provisioning-2'/>
+			<model type='virtio'/>
+		</interface>
+		<interface type='bridge'>
+			<mac address='00:5c:52:31:3b:ad'/>
+			<source bridge='baremetal'/>
+			<model type='virtio'/>
+		</interface>
+		<console type='pty'/>
+		<input type='mouse' bus='ps2'/>
+		<graphics type='vnc' port='-1' autoport='yes'/>
+		<video>
+			<model type='cirrus' vram='9216' heads='1'/>
+		</video>
+	</devices>
+</domain>
+EOF
+virsh define node-1.xml
+virsh define node-2.xml
+# install minikube
+curl -LO https://storage.googleapis.com/minikube/releases/v1.25.2/minikube-linux-amd64
+install minikube-linux-amd64 /usr/local/bin/minikube
+# Install kubectl
+curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl"
+sudo install -o root -g root -m 0755 kubectl /usr/local/bin/kubectl
```

Comment:
nit: unnecessary sudo since the whole script is run with sudo.
```suggestion
install -o root -g root -m 0755 kubectl /usr/local/bin/kubectl
```